src/Adapters/TraceListener.Shared/ApplicationInsightsTraceListener.cs

```diff
@@ -47,6 +47,16 @@ public ApplicationInsightsTraceListener(string instrumentationKey)
             this.TelemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("sd:");
         }
 
+        /// <summary>
+        /// Initializes a new instance of the ApplicationInsightsTraceListener class, specifying a TelemetryClient to use
+        /// </summary>
+        /// <param name="telemetryClient">TelemetryClient to use for tracing</param>
+        public ApplicationInsightsTraceListener(TelemetryClient telemetryClient)
```

Comment:
We typically initialize modules with `TelemetryConfiguration` like [this](https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/develop/Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs#L76) Will this work for your scenario? Or you need `TelemetryClient` specifically?